samples/matter/lock/src/bolt_lock_manager.cpp

```diff
@@ -6,17 +6,62 @@
 
 #include "bolt_lock_manager.h"
 
+#ifdef CONFIG_CHIP_NUS
+#include "bt_nus_service.h"
+#endif
+
 #include "app_event.h"
 #include "app_task.h"
 
 using namespace chip;
 
 BoltLockManager BoltLockManager::sLock;
 
+#ifdef CONFIG_CHIP_NUS
+namespace
+{
+constexpr uint16_t kAdvertisingIntervalMin = 400;
+constexpr uint16_t kAdvertisingIntervalMax = 500;
+constexpr uint8_t kLockNUSPriority = 2;
+constexpr char kLockNUSName[] = "MatterLock_NUS";
```

Comment:
Do you think we could concatenate CONFIG_BT_DEVICE_NAME with "_NUS"? I mean, if customer will change the name to eg. "YaleLock" it could be nice to create "YaleLock_NUS" automatically.

Reply:
yeah, sounds great!